pkgs/games/moon-buggy/default.nix

```diff
@@ -0,0 +1,30 @@
+{stdenv, fetchurl, ncurses}:
+let
+  s = 
+  rec {
+    baseName = "moon-buggy";
+    version = "1.0.51";
+    name = "${baseName}-${version}";
+    url = "http://m.seehuhn.de/programs/moon-buggy-1.0.51.tar.gz";
+    sha256 = "352dc16ccae4c66f1e87ab071e6a4ebeb94ff4e4f744ce1b12a769d02fe5d23f";
+  };
+  buildInputs = [
+    ncurses
+  ];
+in
+stdenv.mkDerivation {
+  inherit (s) name version;
+  inherit buildInputs;
+  src = fetchurl {
+    inherit (s) url sha256;
+  };
```

Comment:
You can actually just inline all attributes instead of using `inherit`. It does not really provide a benefit here. Just add `rec` after `mkDerivation`:

```
stdenv.mkDerivation rec {
```